app/controllers/application_controller.rb

```diff
@@ -5,6 +5,7 @@ class ApplicationController < ActionController::Base
   include TradeTariffFrontend::ViewContext::Controller
   include ApplicationHelper
 
+  before_action :set_enable_service_switch_banner_in_action
```

Comment:
Opted for globally setting the view instance variable to true rather than checking if the value is nil in the helper as this felt more obvious and didn't require treating nil (the default value of an instance variable that hasn't been set) as truthy. Happy to just treat nil as truthy in the helper and no introduce this hook if your prefer.